docs/pages/generic-mixins.rst

```diff
@@ -13,7 +13,8 @@ template.
 
 The following view parameters are supported:
 
-- ``table_class`` –- the table class to use, e.g. ``SimpleTable``
+- ``table_class`` –- the table class to use, e.g. ``SimpleTable``, if not specfied
+  a default table will be provided.
```

Comment:
This sentence needs to mention the default table will be based on `self.model` and having a `self.model` defined is required.